auth_jwt_test/README.rst

```diff
@@ -0,0 +1,117 @@
+=============
```

Comment:
this module should be better called `auth_jwt_example` no?

Reply:
It serves as example but also to test things that are not testable in the base module. I can rename if you prefer, tell me.

Reply:
if it is for tests, it should be called `test_*`. Disclaimer: this is the convention... I'm not against your name :wink:

Reply:
Ok, it has more demo than test stuff so I'll rename to `auth_jwt_demo`.